docs/01-overview/main-areas/telemetry/telemetry-03-traces.md

```diff
@@ -170,10 +188,74 @@ stringData:
   password: XXX
 ```
 
-### Step 5: Rotate the Secret
+### Step 6: Rotate the Secret
 
 As used in the previous step, a Secret referenced with the **secretKeyRef** construct can be rotated manually or automatically. For automatic rotation, update the Secret's actual values and keep the Secret's keys stable. TracePipeline watches the referenced Secrets and detects changes, so the Secret rotation takes immediate effect. When using a Secret owned by the [SAP BTP Operator](https://github.com/SAP/sap-btp-service-operator), you can configure `credentialsRotationPolicy` with a specific `rotationFrequency` to achieve an automated rotation.
 
+## Kyma Components with tracing capabilities
+
+Kyma bundles several modules which are potentially involved in user flows and with must propagate the trace context properly and optional provide custom spans.
+
+### Istio
+
+The Istio module plays a crucial aspect in a distributed trace as the module provides the ingress gateway where usually requests from external are entering the cluster scope and will be enriched with trace context if that did not happen yet. Furthermore, every components being part of the Istio Service Mesh is running an Istio Proxies which propagates the context properly but also creates span data. Having Istio tracing activated and doing trace propagation in your application will already assure that you will get a complete picture of a trace as every component will contribute span data automatically.
+
+The Istio module is configured with an [ExtensionProvider](https://istio.io/latest/docs/tasks/observability/telemetry/) called `kyma-traces`. The provider can be activated on global mesh label using the Istio [Telemetry API](https://istio.io/latest/docs/reference/config/telemetry/#Tracing) by placing a resource to the istio-system namespace like that:
+
+```yaml
+apiVersion: telemetry.istio.io/v1alpha1
+kind: Telemetry
+metadata:
+  name: tracing-default
+  namespace: istio-system
+spec:
+  tracing:
+  - providers:
+    - name: "kyma-traces"
+```
+That will configure all Istio proxies with the `kyma-traces` extension provider which by default will report span data to the trace collector of the telemetry module.
+
+Be aware of, that by default the samplingRate is configured to 1 percent. That means that only 1 trace out of 100 traces will be reported to the trace collector, all others will be dropped. Hereby, the sampling decision itself gets propagated as part of the [trace context](https://www.w3.org/TR/trace-context/#sampled-flag) so that either all involved components are reporting the span data of a trace or none. Turning up the sampling rate will increase the network utilization in the cluster a lot and also will increase the amount of data send to your tracing backend. Usually a very low percentage around 5% are getting used in a productive setup to reduce costs and performance impacts.
```

Comment:
```suggestion
Be aware that by default, `samplingRate` is configured to 1 percent. That means that only 1 trace out of 100 traces is reported to the trace collector, and all others are dropped. Hereby, the sampling decision itself gets propagated as part of the [trace context](https://www.w3.org/TR/trace-context/#sampled-flag) so that either all involved components are reporting the span data of a trace or none. Increasing the sampling rate results in much higher network utilization in the cluster and also increases the amount of data sent to your tracing backend. Usually, a very low percentage of around 5% is used in a production setup to reduce costs and performance impacts.
```